### ISSUE TITLE
Course Information

### ISSUE THREAD

**Issue by icelandgrecian** [microsoftgraph/microsoft-graph-docs#3913]
Hi - Is there any thoughts on adding in course information into this model? If we look at the groups we can see extension data containing course information (CourseSubject, CourseNumber, CourseDescription) which has been loaded in from SDS and it would be useful to pull this information out from educationClass type

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dbea81c1-d612-3ebf-d3ed-01a3c6753108
* Version Independent ID: 855a0093-2991-5e22-4799-d8ea8ee60748
* Content: [educationClass resource type - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/resources/educationclass?view=graph-rest-beta)
* Content Source: [api-reference/beta/resources/educationclass.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/resources/educationclass.md)
* Product: **education**
* GitHub Login: @mmast-msft
* Microsoft Alias: **jthake-msft**

**Comment by icelandgrecian** [comment]
I just tried https://graph.microsoft.com/beta/education/classes/{classId} on a class where the subject was set in Microsoft Teams rather than in SDS. Should this show the subject that is set in Microsoft Teams?

**Comment by mmast-msft** [comment]
No.  The Grade Level and Subject on class team are not represented in Graph.
